dynamometer-workload/src/test/java/com/linkedin/dynamometer/workloadgenerator/TestWorkloadGenerator.java

```diff
@@ -113,10 +112,9 @@ private void testAuditWorkloadWithOutput(String auditOutputPath) throws Exceptio
     assertFalse(dfs.exists(new Path("/denied")));
 
     assertTrue(dfs.exists(new Path(auditOutputPath)));
-    FSDataInputStream auditOutput = dfs.open(new Path(auditOutputPath + "/part-r-00000"));
-    byte[] buf = new byte[auditOutput.available()];
-    assertTrue(auditOutput.read(buf) > 0);
-    System.out.println(new String(buf));
-    assertTrue(new String(buf).matches(".*hdfs,WRITE\\t[0-9]+\\n.*"));
+    try (FSDataInputStream auditOutputFile = dfs.open(new Path(auditOutputPath + "/part-r-00000"))) {
```

Comment:
Minor nit: the correct way to make a child path is `new Path(auditOutputPath, "part-r-0000")`

Reply:
Ah I noticed this one too and forgot to change it 👍 fixed